fieldmanager.php

```diff
@@ -74,6 +74,18 @@ function fieldmanager_load_class( $class ) {
 		return fieldmanager_load_file( 'util/class-fieldmanager-util-' . $class_id . '.php' );
 	}
 
+	/**
+	 * Autoload support for Parsedown.
+	 *
+	 * This is done specifically for Parsedown because it's a third party
+	 * library, which presents a different folder structure than other
+	 * autoloads. If other libraries are introduced in the future, please
+	 * abstract/refecator/generalize this functionality as needed.
+	 */
+	if ( 'Fieldmanager_Parsedown' === $class ) {
+		return fieldmanager_load_file( 'libraries/parsedown/class-fieldmanager-parsedown.php' );
+	}
```

Comment:
I would favor the FQN being `\Fieldmanager\Libraries\Parsedown\Parsedown` and this condition could then be updated to check if the namespace starts with `Fieldmanager\Libraries\`, and if so, load the file in `libraries/**/class-*.php` based on the rest of the namespace and the class name. this would support us adding other libraries in the future. No time like the present to adopt newer standards!